scripts/nsls2-tag-build.py

```diff
@@ -0,0 +1,70 @@
+import os
+import datetime
+import docker
+from pathlib import Path
+
+
+def run_container(*, pkg_name,
+                  docker_image='mrakitin/debian-with-miniconda:latest',
```

Comment:
Adding this kwarg eventually requires to pass it to the Dask's submitter:
```py
packages = ['bluesky', 'databroker', 'ophyd']
statuses = []
for p in packages:
    st = client.submit(run_container, pkg_name=p, pythons=('3.6',),
                       docker_image='mrakitin/debian-with-miniconda:latest'
                       key=f'{p}-{new_ts()}')
    statuses.append(st)
```

Reply:
we can ignore it here. or set as default?

Reply:
No, Dask complains about it if it's not passed, not sure why. Something like:
```py
distributed.worker - WARNING -  Compute Failed
Function:  run_container
args:      ()
kwargs:    {'pkg_name': 'ophyd'}
Exception: TypeError("run_container() missing 1 required keyword-only argument: 'pythons'",)
```

Reply:
then we can remove docker_image here.

Reply:
Yes, it would be reasonable to have it hard-coded to avoid typing it every time.